### ISSUE TITLE
quickget unable to download debian 11.3.0

### ISSUE THREAD

**Issue by ymauray** [quickemu-project/quickemu#567]
#### Expected behaviour

quickget fetched debian 11.3.0

#### Actual behaviour

quickget fails to download the iso

#### Steps to reproduce the behaviour

run `quickget debian 11.3.0 standard`

#### Quickemu output

```
10/08 19:47:08 [ERROR] CUID#7 - Download aborted. URI=https://cdimage.debian.org/debian-cd/11.3.0-live/amd64/iso-hybrid/debian-live-11.3.0-amd64-standard.iso
Exception: [AbstractCommand.cc:351] errorCode=3 URI=https://cdimage.debian.org/debian-cd/11.3.0-live/amd64/iso-hybrid/debian-live-11.3.0-amd64-standard.iso
  -> [HttpSkipResponseCommand.cc:218] errorCode=3 Resource not found
```

#### Linux Distribution & Kernel

No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 22.04.1 LTS
Release:	22.04
Codename:	jammy

Linux heimdall 5.15.0-48-lowlatency #54-Ubuntu SMP PREEMPT Wed Aug 31 12:53:08 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux

**Comment by philclifford** [comment]
I have tried with 4.3 release as installed, 4.3 tag from github and HEAD of master from github and all begin to download fine for 11.3.0  (and 11.4 and 11.5 ...)
I added this recently (pre 4.3)  so that when 11.6 is released and 11.5 moves to `../cdimage/archive/..` it works it out dynamically.



```
function releases_debian() {
      DEBCURRENT=$(wget -q https://cdimage.debian.org/debian-cd/ -O- |grep '\.[0-9]/'|cut -d\> -f9|cut -d\/ -f1)
      local DEBOLD=$(wget -q https://cdimage.debian.org/cdimage/archive/ -O- |grep -e '>[1-9][0-9]\.'|grep -v 'live' | cut -d\> -f9|cut -d\/ -f1 )

     echo  ${DEBOLD} ${DEBCURRENT}
 }
```
Still working for me, and is getting 11.3.0 standard from https://cdimage.debian.org/cdimage/archive/  not https://cdimage.debian.org/debian-cd/  which should fail as above.

The error above saying "Exception: [AbstractCommand.cc:351]" suggests maybe aria2 is in use - could there be an issue with it having a cache from when it downloaded/started downloading 11.3.0 standard when that was the current release ?

Could you try without aria ( e.g. `sudo mv   ${command -v aria2c) ${command -v aria2c) .disabled` ) ?

**Comment by ymauray** [comment]
Crisis averted, panic over, everyone return to your duties. Problem was between the chair and the ppa's .source file, that had been de-activated while migrating from `impish` to `jammy`. All good now. Have a nice week-end, everyone !
